### ISSUE TITLE
rdiodj can't play playlists

### ISSUE THREAD

**Issue by timball** [dasevilla/rdiodj#22]
maybe it's a feature ? but can't add my playlist to the thing ... probably a good thing though ? maybe ? i don't know . just spit ballin' here .

--timball

**Comment by BCWilsonDotCom** [comment]
Do we really want this? Seems like it would be easy to pop into a room and throw in a huge playlist that may throw the currently theme into chaos. Personally, I'd rather not allow adding an entire playlist, but let this tool be a social playlist building app.

**Comment by jackdbernier** [comment]
We are talking about different types of room. From chaotic parties to dj sesions. Maybe that feature could be added to a specific type of room.

If only one person is responsible to add songs in a party, that feature would make sense.

**Comment by MikeNGarrett** [comment]
If the preference is to continue having a listening room style experience, I think playlist support should be added into the song selection process, but as another way to find individual tracks.

**Comment by iangilman** [comment]
@MikeNGarrett Agreed...being able to pull up a playlist and pick a song out of it, rather than having to search for that song directly, would be a great way to be able to curate themed songs for different kinds of parties.

**Comment by aaroncohen** [comment]
It would probably make sense to just allow adding playlists when there was nothing in the queue.
